src/main/resources/apps/pantheon/templates/haml

```diff
@@ -0,0 +1 @@
+../../../../../../../rhdocs-templates/haml
```

Comment:
same comment as before about symlinks.

These templates and stylesheets are a dependency for the project, but how to manage them?

Reply:
I suggest we resolve this thread with the understanding that the original one applies to both symlinks.